src/scripts/install.py

```diff
@@ -98,12 +111,14 @@ def main(args = None):
         exe_mode &= (umask ^ 0o777)
 
     def copy_file(src, dst):
+        dst = prepend_destdir(dst)
         logging.debug('Copying %s to %s' % (src, dst))
         shutil.copyfile(src, dst)
+        return dst
 
     def copy_executable(src, dst):
-        logging.debug('Copying %s to %s' % (src, dst))
-        copy_file(src, dst)
+        dst = copy_file(src, dst)
+        logging.debug('Make %s to %s executable' % (src, dst))
```

Comment:
The `src` variable belongs to the copy step but not to the `chmod +x` step, so I'd just log `dst`